src/components/DBInfo/DBInfo.jsx

```diff
@@ -1,13 +1,13 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 
 class DBInfo extends Component {
-  constructor(props) {
-    super(props);
-  }
+    constructor(props) {
+        super(props);
```

Comment:
What standards for spacing are we using

Reply:
I've been using webstorm, so the defaults there is 4 space tabs